rskj-core/src/main/java/co/rsk/rpc/Web3EthModule.java

```diff
@@ -37,6 +38,10 @@ default String eth_sign(String addr, String data) {
         return getEthModule().sign(addr, data);
     }
 
+    default String eth_signTypedData(String addr, JsonNode data) {
```

Comment:
Out of scope: i think we should decouple this interface from its logic. I'd use an abstract class + interface, instead of using interfaces as abstract classes

Reply:
Agree. Maybe for future work.